mirage/config.js

```diff
@@ -179,7 +262,7 @@ export default function configure() {
     let params = req.queryParams;
     let type = params['filter[type]'];
     let selected = params['filter[selected]'];
-    let filtered = pkg.name && pkg.name.toLowerCase().includes(query);
+    let filtered = pkg.name && query.split(' ').some(w => pkg.name.toLowerCase().includes(w));
```

Comment:
Maybe not important for this PR, but I think `titles` should also have this behavior. It just might get a little hairy with how we can search by different fields for titles. 

Could probably break this out into yet another helper that looks something like `includesWords(string, words)` where `string` could be `pkg.name`, `title.publisherName`, etc., and `words` could be the actual search string

Reply:
Helper function makes sense - I'd like to defer that until the changes are made for other search types. Title being most tricky with multiple search fields. I opted to hold off on the changes for title search since there are 2 more tickets (UIEH-119 and UIEH-121) to address package and title sort. Changing title search (with multiple words)  will most likely impact associated unit tests and I wanted to close out the provider functionality in this PR first